alert/alert.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://utilities91.netlify.app/styles.css" />
+    <link rel="stylesheet" href="./alert.css" />

```

Comment:
Just one suggestion, what you can do that is, you can maintain one single module like a main.css and you can include this all css files in it and say you are using font-awesome. Include it in one place of the project

Reply:
Yes, I am going to do that only, this is only for review purposes, after that, I will put all my components CSS in one file and will import in the main styles.css.